package sakila.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import sakila.model.City;

@Repository
public interface CityRepository extends JpaRepository<City, Short>{

}
